pos_pricelist/models/account_fiscal_position.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class AccountFiscalPositionTax(models.Model):
+    _inherit = "account.fiscal.position.tax"
+
+    company_id = fields.Many2one(
+        related="position_id.company_id", string="Company")
```

Comment:
It's not stored by default, and I have made it on purpose, because the direct access to this field is very strange, so I didn't want to put it as another regular field to be stored. But I can put it if you think so. I'm not sure if when accesing through the one2many field, the security rule is also applied.

Security rules are applied without problem anyway.